File: examples/DropWizardBasicsJava/src/main/java/com/instil/DemoConfiguration.java
package com.instil;

import io.dropwizard.Configuration;

public class DemoConfiguration extends Configuration {
    public String clientName;

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }
}

File: examples/DropWizardExtraKotlin/src/main/kotlin/com/instil/DemoConfiguration.kt
package com.instil

import com.fasterxml.jackson.annotation.JsonProperty
import io.dropwizard.Configuration
import io.dropwizard.client.JerseyClientConfiguration

class DemoConfiguration : Configuration() {

    @get:JsonProperty
    @set:JsonProperty
    var clientName = "Megacorp"

    @get:JsonProperty("jerseyClient")
    @set:JsonProperty("jerseyClient")
    var jerseyClient = JerseyClientConfiguration()
}
